TravelAgency-persistence/src/main/java/cz/muni/fi/travelAgency/dao/ExcursionDaoImpl.java

```diff
@@ -15,6 +16,7 @@
  * @author Rajivv
  */
 @Repository
+@Transactional
```

Comment:
i was create new search method for excursion, it's  find by destination in the file i sent to fb, pls use that file. thx